metaBuild.sh

```diff
@@ -10,10 +10,13 @@ fi
 
 if [ "$1" = "build" ]; then
   SYSTEM_NAME=`uname -s` 
-  if [ "$SYSTEM_NAME" = "Darwin" ]; then
+  if [ "$SYSTEM_NAME" = "Darwin" -o "$SYSTEM_NAME" = "Linux" ]; then
     echo "Building for OSX, architecture set to $ARCH"
+    if [ "$SYSTEM_NAME" = "Dwaring" ]; then
```

Comment:
`Dwaring` looks like a typo, but otherwise this all looks good. I'll do a checkout and test tomorrow.